src/main/java/org/toradocu/translator/Matcher.java

```diff
@@ -85,8 +85,12 @@ public Matcher(int editDistanceThreshold) {
   private Set<CodeElement<?>> filterMatchingCodeElements(
       String filter, Set<CodeElement<?>> codeElements) {
     Set<CodeElement<?>> minCodeElements = new LinkedHashSet<>();
+    //Handle limit case: filter is a one-letter word.
```

Comment:
I'd comment better this. If the word to match is a one-letter word (or empty string), we look for an exact match.